docs/hydra/guides/migrate-v2.mdx

```diff
@@ -0,0 +1,110 @@
+---
+id: migrate-v2
+title: Migrate from Hydra v1 to v2
```

Comment:
This is not a migration but an upgrade that requires data migration.

Reply:
True, good catch :)